frontend/src/components/Navbar.tsx

```diff
@@ -10,37 +11,49 @@ import { makeStyles } from "@material-ui/core";
 import LanguageSwitcher from "./LanguageSwitcher";
 import { AuthContext } from "../components/AuthProvider";
 import theme from "../util/theme";
+import ArrowIcon from "../images/right-arrow-yellow.svg";
 
-const Navbar = () => {
+const Navbar = (props: any) => {
   const { t } = useTranslation();
   const classes = makeStyles(theme as any)();
   const { userData } = useContext(AuthContext);
 
+  const { match, location, history } = props;
+
+  const [mainPage, setMainPage] = useState(false);
+  const [landingPage, setLandingPage] = useState(false);
+
+  useEffect(() => {
+    if (window.location.pathname == "/loops/find") {
+      setMainPage(true);
+      setLandingPage(false);
+    }
+    if (window.location.pathname == "/home") {
+      setLandingPage(true);
+      setMainPage(false);
+    }
+  }, [props.location.pathname]);
```

Comment:
Code with probably be a bit simpler if:

* You use the "useLocation" hook here to fetch pathname
* And pull the the if-statement outside useEffect

useEffect should only be used for code that has side-effects (like networking, console, io, changing browser state, etc).